databricks/koalas/generic.py

```diff
@@ -77,6 +77,42 @@ def compute(self):
         """Alias of `toPandas()` to mimic dask for easily porting tests."""
         return self.toPandas()
 
+    @derived_from(pd.DataFrame)
+    def fillna(self, value=None, axis=None, inplace=False):
+        """Fill NA/NaN values.
+
+        :param value: scalar, dict, Series
+                    Value to use to fill holes. alternately a dict/Series of values
+                    specifying which value to use for each column.
+                    Dataframe is not supported.
+        :param axis: {0 or `index`}
+                    1 and `columns` are not supported.
+        :param inplace: boolean, default False
+                    Fill in place (do not create a new object)
+        :return: :class:`Series` or :class:`DataFrame`
+        """
+
+        if axis is None:
+            axis = 0
+        if value is not None:
+            if axis == 0 or axis == "index":
+                if isinstance(value, (float, int, str, bool, dict)):
+                    df = self._spark_fillna(value)
```

Comment:
Now that #90 has been merged, it should now directly call the underlying Spark dataframe directly and not through the shim. For dataframes: `self._sdf.na.fill(value)` (if I am correct).